File: UI/eventsappadmin/lib/models/stavke_narudzbe.dart
import 'package:json_annotation/json_annotation.dart';

part 'stavke_narudzbe.g.dart';

@JsonSerializable()
class StavkeNarudzbe {
  int? narudzbaStavkaId;
  int? narudzbaId;
  int? tipKarteId;
  double? cijena;
  int? kolicina;
  String? tipKarte;
  String? dogadjaj;

  StavkeNarudzbe(this.narudzbaStavkaId, this.narudzbaId, this.tipKarteId,
      this.cijena, this.kolicina, this.tipKarte, this.dogadjaj);

  factory StavkeNarudzbe.fromJson(Map<String, dynamic> json) =>
      _$StavkeNarudzbeFromJson(json);

  Map<String, dynamic> toJson() => _$StavkeNarudzbeToJson(this);
}

File: eventsApp.Model/StavkeNarudzbe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eventsApp.Model
{
    public class StavkeNarudzbe
    {
        public int NarudzbaStavkaId { get; set; }

        public int NarudzbaId { get; set; }

        public int TipKarteId { get; set; }

        public int Kolicina { get; set; }

        public decimal Cijena { get; set; }

        public string? TipKarte { get; set; }

        public string? Dogadjaj { get; set; }
    }
}
